tests/runner/iframeTestHarness.js

```diff
@@ -0,0 +1,20 @@
+define('iframeTestHarness', [
+    '$'
+], function(_$) {
```

Comment:
Originally this was inside the test file, where we had both $ from the parent window, and also $ from the iframe. We needed them to be distinct. We don't need this naming anymore, so it can probably be renamed to $.